lib/main.dart

```diff
@@ -24,21 +27,37 @@ class _HelloAppState extends State<HelloApp> {
     super.initState();
     if (MOBILE_SDK_KEY != '') {
       final LDConfig config = LDConfigBuilder(MOBILE_SDK_KEY).build();
-      final LDUser user = LDUserBuilder('user key').build();
+
+      // Set up the user properties. This user should appear on your LaunchDarkly users dashboard
+      // soon after you run the demo.
+      final LDUser user = LDUserBuilder('example-user-key')
+      .name('Sandy');
+      .build();
       LDClient.start(config, user).whenComplete(() {
           updateFlagEvaluation();
           LDClient.registerFeatureFlagListener(FEATURE_FLAG_KEY, updateFlagEvaluation);
-      });
+      }
+      setState(() { message = 'SDK successfully initialized!'});
+      )
     } else {
-      setState(() { message = 'MOBILE_SDK_KEY not set'; });
+      setState(() { message = 'Please edit lib/main.dart to set MOBILE_SDK_KEY to your LaunchDarkly SDK key first'; });
+      setState(() { message = 'SDK failed to initialize'});
```

Comment:
Similar here--couldn't tell if the absent SDK key would cause an initialization failure or not.

Reply:
I had that thought too! I wasn't sure how to initiate a message appended to another, though.